network/loss.md

```diff
@@ -0,0 +1,102 @@
+# Test Network Chaos on TiDB
+
+## Network Loss
+
+### Description
+
+Test the availability of TiDB cluster in network package loss scenarios.
```

Comment:
The above are all "the TiDB", and this should also be changed to be consistent with the above?

Reply:
done, PTAL again